multi-patchs.tex

```diff
@@ -198,8 +291,8 @@ \section{Advection equation}
 \end{aligned}
 \right.
 \end{equation}
-where $\bar{\psi}$ is a discrete flow computed with
-a time integration method. In the case of Runge-Kutta method, the advection field needs to be evaluated at intermediate feet.
+
+with a time integration method. In the case of Runge-Kutta method, the advection field needs to be evaluated at intermediate feet.
```

Comment:
Why did you delete the mention of the discrete flow? $\bar{\psi}$ is still undefined